### ISSUE TITLE
Proposed revision to the Note: at the top of the Onix section

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#9]
From Madeleine's email: https://lists.w3.org/Archives/Public/public-epub3/2020Feb/0004.html

ONIX does not have an exact 1:1 mapping with EPUB accessibility metadata so unfortunately not all of the accessibility metadata found in an EPUB exists in ONIX at the time of this publication. There are plans to add this metadata to future versions of ONIX but no time frame has been announced. This EPUB to ONIX crosswalk outlines the current overlap in metadata which will get updated as these two specifications evolve. [add Some Schema.org metadata properties, referenced in EPUB, are descriptions of reading systems and not individual books; this means they will not be appropriate for use in ONIX and will not have a crosswalk. This includes the property "bookmarks" which indicates the ability of a reading system to allow users to set and navigate to bookmarks.] It is important to note that there were a number of new accessibility metadata codes added to ONIX 3 to support the Accessibility 1.0 specification. [delete Which means that] ONIX 2 has a limited number of accessibility metadata codes and is [delete something] not covered in this document.

**Comment by madeleinerothberg** [comment]
Proposed text for each techniques doc to distinguish book features from reading system features

ONIX
Some Schema.org accessibility metadata properties are descriptions of reading systems and not individual publications; this means they will not be appropriate for use in ONIX. This includes the property "bookmarks," which indicates the ability of a reading system to allow users to set and navigate to bookmarks. In addition, some kinds of transformations described in Schema.org accessibility metadata are disallowed if a book is constructed to prevent transformations, but can only be provided if the reading system offers users control of that feature. Those features therefore cannot be declared present in the publication's metadata because not all reading systems that support the publication file will provide that feature. 

EPUB
Same text but substitute "inside the EPUB package" for "in ONIX"

Additionally, the paragraph mentioned in the original issue still needs tweaks to soften the language about what ONIX does and doesn't do and to avoid implying timelines. Proposed new text for that, without any language about bookmarks feature since that will be elsewhere.

ONIX does not have a 1:1 mapping with EPUB or Schema.org accessibility metadata. Some accessibility assertions are made differently and some may not currently be possible. This EPUB to ONIX crosswalk [link] outlines the current overlap in metadata and will be updated as these two specifications evolve. It is important to note that ONIX 3 includes a number of new accessibility metadata codes. ONIX 2 has more limited accessibility metadata and is not covered in this document.
